StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityFlowView.swift

```diff
@@ -231,12 +244,17 @@ extension IdentityFlowView {
 
     fileprivate func installConstraints() {
         buttonBackgroundView.translatesAutoresizingMaskIntoConstraints = false
+        buttonTopBackgroundView.translatesAutoresizingMaskIntoConstraints = false
         buttonTopContentView.translatesAutoresizingMaskIntoConstraints = false
 
         NSLayoutConstraint.activate([
             // Constrain buttonTop top of buttons
-            buttonTopContentView.centerXAnchor.constraint(equalTo: centerXAnchor),
-            buttonTopContentView.bottomAnchor.constraint(equalTo: buttonBackgroundView.topAnchor, constant: -Style.buttonTopBottomPadding),
+            buttonTopBackgroundView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            buttonTopBackgroundView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            buttonTopBackgroundView.bottomAnchor.constraint(equalTo: buttonBackgroundView.topAnchor),
+            buttonTopContentView.centerXAnchor.constraint(equalTo: buttonTopBackgroundView.centerXAnchor),
```

Comment:
not sure if I am understanding the layout correctly but adding a note just in case: `centerXAnchor` will center but will likely not handle the layout properly if `buttonTopContentView` width is greater than its parent (it might clip?)

Reply:
yes, but this field is always `Stripe Privacy Policy`